titan-compiler/lexer.lua

```diff
@@ -22,7 +22,10 @@ local C, Cb, Cg, Ct, Cmt, Cc = lpeg.C, lpeg.Cb, lpeg.Cg, lpeg.Ct, lpeg.Cmt, lpeg
 local T = lpeg.T
 
 local syntax_errors = require "titan-compiler.syntax_errors"
-local labels = syntax_errors.label_to_int
+local labels = {}
```

Comment:
We don't need this table anymore. Couldn't we just remove this piece of code and use the actual label now?
`s/labels.\([A-Za-z_]*\)/"\1"/g`

Reply:
I can fix this in a subsequent pull request. If this is the only problem I think we can merge it now.